scripts/count-tiles-without-ids.ts

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/node
+
+// to run this script on Node 20
+// $ cd scripts
+// $ npx tsx count-tiles-without-ids.ts
+// I think this will also work on Node 16 and higher.
+
+import fs from "fs";
+import path from "path";
+
+// Taken from here:
+// https://stackoverflow.com/a/45130990
+async function* getFiles(dir) {
+  const dirents = await fs.promises.readdir(dir, { withFileTypes: true });
+  for (const dirent of dirents) {
+    const res = path.resolve(dir, dirent.name);
+    if (dirent.isDirectory()) {
+      yield* getFiles(res);
+    } else {
+      yield res;
+    }
+  }
+}
```

Comment:
I'd move this into a separate utility file.

Reply:
Since I was having problems importing local modules, I'll leave it here. But if you are able to import local modules in your new scripts let me know.